### PR TITLE
Bearification

### LINKED ISSUE #3: Not enough bears.
I really think we're lacking bear features.

I'd like to propose bear-unique branches, such as https://github.com/mnzt/a-bear/tree/exotic.

### PR DESCRIPTION
**Pull request by tobbbles**
Closes #3.

### CODE REVIEW

**Comment by necenzurat** [comment]
Would be nice if they were the same size.

File: index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 const fs = require('fs')
 
-function bear () {
-  return fs.readFile('bear.jpg', (err, data) => {
+exports = module.exports = function (bearType) {
```
Comment (RichardLitt) [inline index.js:4]: default to polar if no bear is given?

Comment (RichardLitt) [inline index.js:4]: Otherwise this is breaking. Wouldn't want any builds to break.

Comment (tobbbles) [inline index.js:4]: :+1:

**Comment by RichardLitt** [comment]
I rebased these for you and squashed them. Landed in https://github.com/RichardLitt/a-bear/commit/2636b100cab29b6439fbe26f939fbfd7e707ba9e. Thanks!

**Pull request closed without merging**